components/admin/admin-touch-interactions.tsx

```
@@ -0,0 +1,342 @@
+  return (
+    <div className={cn("space-y-1", className)}>
+      {React.Children.map(children, (child, index) => (
+        <div
+          key={index}
+          onClick={() => handleItemPress(index)}
+          className={cn(
+            "cursor-pointer transition-all duration-150",
+            pressedIndex === index ? "scale-95 opacity-80" : "",
+            "active:scale-95 active:opacity-80"
+          )}
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)